src/main/webapp/index.html

```diff
@@ -1,26 +1,24 @@
-<!DOCTYPE html>
 
+<!DOCTYPE html>
 <html>
-
-<head>
-	<meta charset="UTF-8">
-	<title>Givr</title>
-	<link rel="stylesheet" href="style.css">
-	<script src="index_script.js"></script>
-  <script src="user.js"></script>
-</head>
+  <head>
+    <meta charset="UTF-8">
+    <title>Givr</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="index_script.js"></script>
```

Comment:
The script tag for file user.js seems to have been taken out.

Reply:
this was because an accidental merging error on my part